app/models/room.rb

```diff
@@ -7,7 +7,7 @@ class Room < SearchableRecord
   has_and_belongs_to_many :chairs
   has_and_belongs_to_many :tags
   has_many :people, dependent: :nullify
-  belongs_to :building, optional: true # optional, because rooms should be creatable without creating a building
+  belongs_to :floor, optional: true # optional, because rooms should be creatable without creating a floor
```

Comment:
```suggestion
  belongs_to :floor
```
I would remove optional, there cannot be a floor outside of a building (just assure the building is created before the floor)